scivalidation.tex

```diff
@@ -0,0 +1,193 @@
+especially the larger "dress rehearsals" and "data challenges" as
+listed in Section~\ref{sect:schedule} (Master Schedule). 
+
+Following successful service stand-up or data challenge execution (at
+integration and unit test level), the generated data products or integrated
+services are turned over to the SV team.  The SV team performs additional
+tests and data analyses to exercise the integrated system and assess its
+quality relative to expectations for the current phase of construction. 
+This assessment is fed back to DM Subsystem Science and Systems Engineering
+teams to inform them about the status and needed improvements to the system.
+
+Beyond reporting on the results, the SV team examines the tests or
+procedures developed in this phase and identifies those that are good new
+metrics of system quality and could be run in an automated fashon.  These
+are fed back to the development teams for productizing and incorporation
+into the automated QC systems.
+
+\subsection{Deliverables}
+
+Key deliverables of Science Validation activities are:
+\begin{itemize}
+
+\item Reports on the assessed capability of the Data Management System to
+satisfy stakeholder needs.  The assessments shall take into account the
+expected maturity of the system being tested.
+
+\item Recommendations for improvements and changes, both in the quality of
+as-constructed systems (i.e., what needs to be built differently or better,
+to make it more consistent with the system vision), as well as the overall
+system vision (i.e., recommendations on where the vision may need to be
+modified to fully respond to stakeholder needs).
+
+\item Measurements of performance metrics that do not lend themselves to
+easy automation (e.g., science activities requiring human involvement, like
+visual classification, or UX tests).
+
+\item Identification of new performance metrics to be tracked, including
+potential deliveries of code to the DM Construction and I\&T teams for
+inclusion in automated quality control pipelines.
+
+\item Other deliverables as charged when chartering a particular SV exercise.
+
+\end{itemize}
+
+\subsection{Organization and Resources}
+
+\begin{figure}
+\centering
+\scalebox{0.5}{\hskip 0.0in \includegraphics[trim={2.4cm 3cm 2.4cm
+3cm},clip,page=2]{figures/dm-subsystem-science.pdf}}
+\caption{Organogram of the Data Management Science Validation Team. 
+The group is chaired by the DM Science Validation Scientist,
+with the DM Science Pipelines Scientist and Institutional Science Leads making up
+the permanent membership. Depending on the SV activities being executed at any
+given time, the group may draw on additional temporary members from DM SST Staff,
+the broader DM Construction staff, as well as external scientists (e.g.,
+Science Collaboration members committed to assisting SV goals). SV membership
+is reassessed on a cycle by cycle basis, with estimates incorporated in the
+long-term plan.
+\label{fig:DMsvg}}
```

Comment:
Figure is never referenced from the text. I think the bulk of this caption should be integrated into the main body of text (and much of it already is). What do the colors of the boxes mean?